### PR TITLE
Update to Meilisearch v1.1

### LINKED ISSUE #11: Make the price range facet dynamic
https://github.com/meilisearch/product/discussions/117#discussioncomment-4981262

as soon as https://github.com/meilisearch/meilisearch/milestone/45 will be released ?

### LINKED ISSUE #12: Add favicon
Unless we have a better idea, I think a simple Meilisearch logo would be good

### PR DESCRIPTION
**Pull request by Strift**
## What does this PR does?

- Add default props on PageNumber.vue
- Fixes #11 
- Fixes #12 
- Bump meilisearch to v0.32.2
- Bump @meilisearch/instant-meilisearch to v0.12.0

### CODE REVIEW

File: components/organisms/MeiliSearchRangeFilter.vue [hunk 1]
```diff
@@ -24,11 +21,7 @@ const toValue = (currentValue: Range, boundaries: Range): [number, number] => {
 </script>
 
 <template>
-  <AisRangeInput
-    :attribute="attribute"
-    :min="minPrice"
-    :max="maxPrice"
-  >
+  <AisRangeInput :attribute="attribute">
```
Comment (bidoubiwa) [inline components/organisms/MeiliSearchRangeFilter.vue:24]: ✨

File: package.json [hunk 2]
```diff
@@ -44,6 +44,6 @@
   },
   "dependencies": {
     "dotenv": "^16.0.3",
-    "meilisearch": "^0.31.1"
+    "meilisearch": "0.32.2"
```
Comment (bidoubiwa) [inline package.json:47]: I know it technically doesn't change anything and at the end of the day on apps you chose where you want to put your dependencies, but out of curiosity, why is one in `devDependencies` and the other in `dependencies` ?

**Review by bidoubiwa — APPROVED** [review]
LGTM 🔥🔥🔥

Just a small question but it's not an issue

File: package.json (hunk 2 above)
Comment (Strift) [reply package.json:47]: I build the Nuxt 3 app as a static site. So I see everything used to build the Nuxt app as a dev dependencies: they get bundle via Vite but are not installed in the production site.

However, should you want to run the `setup` script, you really need to install meilisearch and dotenv. In a way, these deps are required to manage the production app.

I don't have the strongest opinions on this tbh. I find deps vs devDeps to become more and more vague with the use of build tools.

**Pull request merged**